connect_four.py

```diff
@@ -37,11 +36,14 @@ def check_col(self, col: str) -> int:
         else:
             return col_i - unicode_A
 
-    def insert_new_piece(self, col: int, color: str) -> bool:
-        row = self.num_rows - len(self.columns[col])
+    def insert_new_piece(self, col: int, color) -> int:
+        row = (self.num_rows - 1 - len(self.columns[col]))
+        print(f"1, row is {row}, col is {col}")
         self.columns[col].append(color)
+        print("2")
```

Comment:
Are these prints intentional?

Reply:
My bad, I was debugging and forgot to delete them.